index.html

```diff
@@ -1 +1,46 @@
-<!-- JavaScript課題①-3 -->
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>task1-4</title>
+</head>
+<body>
+
+  <script>
+    const taskList = [
+      {content:'机を片付ける', type:'掃除'},
+      {content:'牛乳を買う', type:'買い物'},
+      {content:'散歩をする', type:'散歩'},
+    ];
+
+    const taskTable = () => {
+    console.log('==========================');
+    console.log('現在持っているタスクの一覧');
+    console.log('==========================');
+
+    Object.keys(taskList).forEach(index => {
+      const table = console.log(`${index} : [内容]${taskList[index].content}、[ジャンル]${taskList[index].type}`)
+    });
```

Comment:
`forEach` だけを使うともっと簡素に記述できます。

```suggestion
    taskList.forEach((task, index) => {
      console.log(`${index} : [内容]${task.content}、[ジャンル]${task.type}`)
    });
```